packages/plugins/plugin-deck/src/components/DeckLayout/ComplementarySidebar.tsx

```diff
@@ -115,28 +130,24 @@ export const ComplementarySidebar = ({ panels, current }: ComplementarySidebarPr
             <ToggleComplementarySidebarButton />
           </div>
         </div>
-        {panels.map((panel) => (
+        {availablePanels.map((panel) => (
           <Tabs.Tabpanel
             key={panel.id}
             value={panel.id}
             classNames='absolute data-[state="inactive"]:-z-[1] inset-block-0 inline-start-0 is-[calc(100%-var(--r0-size))] lg:is-[--r1-size] grid grid-cols-1 grid-rows-[var(--rail-size)_1fr_min-content] pbs-[env(safe-area-inset-top)]'
             {...(layout.complementarySidebarState !== 'expanded' && { inert: 'true' })}
           >
-            {panel.id === activePanelId && node && (
+            {panel.id === activePanelId && data && (
               <>
                 <h2 className='flex items-center pli-2 border-separator border-be'>
                   {toLocalizedString(panel.label, t)}
                 </h2>
                 <ScrollArea.Root>
                   <ScrollArea.Viewport>
                     <Surface
-                      key={activeEntryId}
+                      // key={activeEntryId}
                       role={`complementary--${activePanelId}`}
```

Comment:
Remove?

Reply:
oh yeah, I forgot to test whether this broke anything or not, will do that before I land this